spec/javascript-spec.coffee

```diff
@@ -85,6 +85,13 @@ describe "Javascript grammar", ->
       {tokens} = grammar.tokenizeLine('0O1411')
       expect(tokens[0]).toEqual value: '0O1411', scopes: ['source.js', 'constant.numeric.js']
 
+    it "regular expressions have explicit count modifiers", ->
```

Comment:
Oops, thanks for the review.
Do you have better than `"verifies that regular expressions have explicit count modifiers"`?